libs/transactions/src/lib/transactions-journey.module.ts

```diff
@@ -17,21 +17,28 @@ import { TransactionsJourneyConfiguration } from './services/transactions-journe
 import { TransactionsRouteTitleResolverService } from './services/transactions-route-title-resolver.service';
 import { TransactionsHttpService } from './services/transactions.http.service';
 import { TransactionsViewComponent } from './views/transactions-view/transactions-view.component';
+import { TransactionDetailsComponent } from './views/transaction-details/transaction-details.component';
```

Comment:
In GAS for the all view components we use `-view` suffix

Reply:
done